.github/workflows/zxc-jrs-regression.yaml

```diff
@@ -302,25 +302,23 @@ jobs:
         id: platform-gradle-build
         uses: gradle/gradle-build-action@v2
         with:
-          build-root-directory: platform-sdk
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: assemble --scan
+          arguments: -p platform-sdk assemble --scan
 
       - name: Hedera Node Gradle Assemble
         id: hedera-gradle-build
         uses: gradle/gradle-build-action@v2
         if: ${{ inputs.hedera-tests-enabled }}
         with:
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: assemble --scan
+          arguments: -p platform-sdk assemble --scan
 
       - name: Regression Gradle Assemble
         id: regression-gradle-build
         uses: gradle/gradle-build-action@v2
         with:
-          build-root-directory: platform-sdk/regression
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: assemble --scan --parallel
+          arguments: -p platform-sdk assemble --scan --parallel
```

Comment:
This must be reverted as this is wrong and breaks the CI pipeline. This is building code checked out from the `swirlds/swirlds-platform-regression` repository underneath of `platform-sdk/regression` but this change makes this identical to the step immediately above it.

Reply:
Sorry and good catch. I didn't know about 'regression' and misread this step. Reverted.

Reply:
Confirmed this is resolved.